fossunited/payments/doctype/razorpay_payment/razorpay_payment.py

```diff
@@ -4,6 +4,7 @@
 import frappe
 from frappe.model.document import Document
 
+from fossunited.doctype_ids import EVENT
```

Comment:
```suggestion
from fossunited.doctype_ids import EVENT_DOCTYPE
```

would read better.

Reply:
This was a conscious choice at the time of creating IDs file